firefox-ios/Client/Application/DependencyHelper.swift

```diff
@@ -41,6 +38,9 @@ class DependencyHelper {
         let windowManager: WindowManager = appDelegate.windowManager
         AppContainer.shared.register(service: windowManager)
 
+        let themeManager: ThemeManager = appDelegate.themeManager
+        AppContainer.shared.register(service: themeManager)
+
```

Comment:
curious - why the rearrangement of order?

Reply:
IIRC the reason was that initially I believed we would need to know the first/default window's UUID at some point during the initialization of the the theme manager (which meant that we wanted to have the `WindowManager` initialized first), but I believe this is no longer strictly required because I ended up taking a different approach for some of the refactors which didn't require it. However I think it's still useful to have the WindowManager initialized before anything else because at some point it may end up being useful to be able to ask for the default/main window UUID during init for the other managers.

Reply:
ooo good to know, if it was strictly required, I would prob ask to leave a comment, but in this case, it's good as-is! thank you for explaining the reasoning behind it!